contrib/objectstore/Makefile

```diff
@@ -0,0 +1,11 @@
+GOVENDOR:=${GOPATH}/bin/govendor
```

Comment:
I would call this Makefile from the contrib target, llike the `snort` one:
https://github.com/skydive-project/skydive/blob/master/Makefile#L354

So that we are sure that we won't break it changing internal sutff